.openshift/templates/jenkins-slave-generic-template.j2

```diff
@@ -1,6 +1,6 @@
 ---
 kind: Template
-apiVersion: v1
+apiVersion: template.openshift.io/v1
```

Comment:
Since other changes in this PR is reorganizing `apiVersion` to come before `kind` let's do the same here.